var/spack/repos/builtin/packages/py-melissa-core/package.py

```diff
@@ -37,6 +39,8 @@ class PyMelissaCore(PythonPackage):
     depends_on("py-python-rapidjson@1.8:", type="run")
     depends_on("py-scipy@1.10.0:", type="run")
     depends_on("py-cloudpickle@2.2.0:", type="run")
+    depends_on("py-iterative-stats@0.1.0:", type="run", when="@develop")
+    depends_on("py-iterative-stats@0.0.4:", type="run", when="@joss")
     # requirements_deep_learning.txt (DL with torch)
     depends_on("py-tensorboard@2.10.0:", type="run", when="+torch")
     depends_on("py-matplotlib", type="run", when="+torch")
```

Comment:
Isn't python-hostlist also required for sc23?

Reply:
Indeed, `python-hostlist` is required by the SC23 tag. I while add a Spack package for it.